examples/trials/mnist-tfv2/launch.py

```diff
@@ -0,0 +1,27 @@
+# FIXME: For demonstration only. It should not be here
+
+from pathlib import Path
```

Comment:
Are we releasing this feature as experimental or recommended way to launch an experiment?

Reply:
Experimental I think.

Reply:
@ultmaster  what is the difference? why experimental is better?

Reply:
Suggest full IT and UT pipeline if we are recommending this feature.

Reply:
i think we should add IT and UT

Reply:
according to the completeness, the feature should be experimental. still suggest to add IT and UT in this release